### PR TITLE
[CP-1070] Move release logic to update module

### PR DESCRIPTION
**Pull request by igorbezsmertnyi**
Jira: [CP-1070]

**Description**
Moving releases logic to separated `update` module

<details>
<summary><b>Screenshots</b></summary>
// put images here
</details>

[CP-1070]: https://appnroll.atlassian.net/browse/CP-1070?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/app/src/update/requests/get-release.request.ts [hunk 1]
```diff
@@ -22,11 +20,11 @@ const productsMapper = {
   [DeviceType.MuditaHarmony]: Product.BellHybrid,
 }
 
-const getAllReleases = async (
+export const getRelease = async (
```
Comment (dkarski) [inline packages/app/src/update/requests/get-release.request.ts:23]: why did you change the name when the structure response is the same? 🤔 -> `AllReleasesResponse`

File: packages/app/src/update/listeners/get-all-development-releases.listener.ts [hunk 2]
```diff
@@ -81,9 +37,9 @@ const releasesRequest = async (
   }
 }
 
-const registerGetAllReleasesListener = () => {
+export const registerGetAllDevelopmentReleasesListener = () => {
```
Comment (dkarski) [inline packages/app/src/update/listeners/get-all-development-releases.listener.ts:40]: I'm not sure that all releases are as Development 😛 I think that we could avoid this prefix 🙃

**Comment by dkarski** [comment]
Greate Idea to move release logic to update module 👍 👏

File: packages/app/src/update/listeners/get-all-development-releases.listener.ts (hunk 2 above)
Comment (igorbezsmertnyi) [reply packages/app/src/update/listeners/get-all-development-releases.listener.ts:40]: I guess it would be better to keep these prefix

File: packages/app/src/update/requests/get-release.request.ts (hunk 1 above)
Comment (igorbezsmertnyi) [reply packages/app/src/update/requests/get-release.request.ts]: Fixed

File: packages/app/src/update/requests/get-all-releases.request.ts [hunk 3]
```diff
@@ -22,11 +20,11 @@ const productsMapper = {
   [DeviceType.MuditaHarmony]: Product.BellHybrid,
 }
 
-const getAllReleases = async (
+export const getAllReleases = async (
   deviceType: DeviceType
 ): Promise<AllReleasesResponse> => {
   const releases: Release[] = await ipcRenderer.callMain<undefined, Release[]>(
-    GetAllReleasesEvents.Request
+    IpcUpdate.GetDevelopmentRelease
```
Comment (dkarski) [inline packages/app/src/update/requests/get-all-releases.request.ts:27]: the same case like in the https://github.com/mudita/mudita-center/pull/629#discussion_r802593895

File: packages/app/src/update/listeners/get-all-development-releases.listener.ts (hunk 2 above)
Comment (dkarski) [reply packages/app/src/update/listeners/get-all-development-releases.listener.ts:40]: I'm not sure 😛 this listener will emit all releases not just filtered (by development)? 🤔

File: packages/app/src/update/requests/get-all-releases.request.ts (hunk 3 above)
Comment (igorbezsmertnyi) [reply packages/app/src/update/requests/get-all-releases.request.ts]: https://github.com/mudita/mudita-center/pull/629/commits/8116b41233329e3201474e7769aaab262b95ca20

File: packages/app/src/update/listeners/get-all-development-releases.listener.ts (hunk 2 above)
Comment (igorbezsmertnyi) [reply packages/app/src/update/listeners/get-all-development-releases.listener.ts]: https://github.com/mudita/mudita-center/pull/629/commits/7d4164dffb783556c4508f2bc2006b0aef4d25ef
